templates/geosafe/analysis/dynamic_scripts/create/main.html

```diff
@@ -369,7 +369,13 @@
         $.get('{% url 'geosafe:layer-tiles' %}', {'layer_id': layer_id}, function (data) {
 
             //default use of the marker
-            var marker = L.marker([data['layer_bbox_y0'], data['layer_bbox_x0']], {
+            center_y = data['layer_bbox_y0'] + ((data['layer_bbox_y1'] - data['layer_bbox_y0'] ) / 2);
```

Comment:
just use (y0+y1)/2 mathematically the same :p.

Reply:
yes, you are right 😆